content/responses/metadata_query_index.yml

```diff
@@ -0,0 +1,56 @@
+---
+title: Metadata query index
+
+type: object
+
+x-box-resource-id: metadata_query_index
+
+x-box-tag: search
+
+description: |-
+  A metadata query index
+
+properties:
+  id:
+    type: string
+    example: "-9876"
+    description: The ID of the metadata query index.
+
+  type:
+    type: string
+    description: "`metadata_query_index`"
+    example: metadata_query_index
+    nullable: false
+
+  state:
+    type: string
+    description: "The state of the metadata query index"
+    example: active
+    enum:
+      - building
+      - active
+      - disabled
```

Comment:
what do we do about an enum that we're unable to map to one of these

Reply:
@akhor93 the enum list will just be whatever potential options that may be returned as part of the `state` property. If you just adjust / add / delete from the list then this will build the list appropriately. In short, feel free to adjust the enum list to whatever values it should be.